python/ray/util/sgd/torch/torch_runner.py

```diff
@@ -148,25 +142,31 @@ def _try_setup_apex(self):
             self.models, self.optimizers = amp.initialize(
                 self.models, self.optimizers, **self.apex_args)
 
-    def setup(self):
-        """Initializes the model."""
-        _remind_gpu_usage(self.use_gpu, is_chief=True)
+    def setup_components(self):
```

Comment:
Maybe create a single public `setup()` method in both runner and distributed runner and make `setup_components`, `setup_operator`, and `setup_ddp_and_operator` private?

Reply:
I've converted the setup to be a single command for the runner. Unfortunately, Ray doesn't have a barrier primitive (which is needed for distributed runner before calling ddp), so I've left that one as is.